_posts/2016-08-27-mouse-dna-magnetic-fields.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: Low magnetic fields leave mouse DNA undamaged
+author: RaoOfPhysics
+image:
+  feature: 2016-08-27-mouse-dna-magnetic-fields.jpg
+  credit: Nogas1974
+  caption: “DNA com GGN”
+---
+
+_Cells in brain, kidney and liver, which are involved in the transport, reabsorption or storage of iron, withstand sustained exposure to magnetic fields._
```

Comment:
Really don't think it's necessary for the dek to say anything about iron metabolism.
can withstand